caas/kubernetes/provider/application/application.go

```diff
@@ -1565,6 +1566,49 @@ func (a *app) volumeName(storageName string) string {
 	return fmt.Sprintf("%s-%s", a.name, storageName)
 }
 
+// getPVCNames returns a mapping of volume name to PVC name for this app's PVCs.
+func (a *app) getPVCNames() (map[string]string, error) {
+	// Fetch all Juju PVCs associated with this app
+	opts := metav1.ListOptions{
+		LabelSelector: fmt.Sprintf("app.kubernetes.io/managed-by = juju, app.kubernetes.io/name = %s", a.name),
```

Comment:
Use utils.LabelsToSelector to create this from the github.com/juju/juju/caas/kubernetes/provider/utils package

Reply:
Thanks! Done.